lib/components/result-view/history.js

```diff
@@ -45,7 +45,10 @@ const History = observer(({ store }: { store: OutputStore }) => {
           onClick={store.incrementIndex}
         />
       </div>
-      <div className="multiline-container">
+      <div
+        className="multiline-container"
+        style={{ fontSize: atom.config.get(`Hydrogen.outputAreaFontSize`) }}
```

Comment:
Does this work even without `px`?

In any case we should now handle the missing default value here:
```js
atom.config.get(`Hydrogen.outputAreaFontSize`) || "inherit"
```